### PR TITLE
@eessex => Image size support

### PR DESCRIPTION
**Pull request by kanaabe**
All `ImageCollection` sections will now also provide a width/height. Same for `ImageSet`.

The WIP part is for the conversion process. We'll have to fetch images/artworks when converting Image and Artwork sections to ImageCollection so we can provide the width/height info. I'm playing around with moving this logic to the `Sections` collection where it seems fit a little better...sorry it's taking a while to get this out! Really hate halting other deploys for this kinda thing but 🤷‍♀️

### CODE REVIEW

File: client/apps/edit/client.coffee [hunk 1]
```diff
@@ -13,20 +13,94 @@ EditAdmin = require './components/admin/index.coffee'
...
+
+convertImageSet = (section, callback) ->
+  async.parallel(
+    section.get('images').map (image) -> (cb) ->
+      img = new Image()
+      img.src = image.image or image.url # image or artwork
+      img.onload = ->
+        image.width = img.width
+        image.height = img.height
+        image.artists = [image.artist] if image.type is 'artwork'
```
Comment (eessex) [inline client/apps/edit/client.coffee]: do all the image set artworks just have the singular artist field?

Comment (kanaabe) [inline client/apps/edit/client.coffee]: yeah i was going to do the same conversion process for image sets but i think we said that the frontend doesn't support it yet, right?

File: api/apps/articles/model/schema.coffee [hunk 2]
```diff
@@ -56,6 +58,11 @@ denormalizedArtwork = (->
         name: @string().allow('', null)
         slug: @string().allow('', null)
     ).allow(null).default([])
+    artist: @object().keys
+      name: @string().allow('', null)
+      slug: @string().allow('', null)
```
Comment (kanaabe) [inline api/apps/articles/model/schema.coffee]: Adding back in singular artist for now because it looks like ImageSet is broken on staging

**Comment by kanaabe** [comment]
Merging this!

**Pull request merged**